sw/otbn/crypto/p384_curve_point_valid.s

```diff
@@ -26,25 +26,17 @@ start:
   unimp
 
 validate_point:
+  /* Fill gpp registers with pointers to coordinates */
```

Comment:
nit: I think this is a typo?
```suggestion
  /* Fill gpr registers with pointers to coordinates */
```

There are a few more repetitions of the same comment below.